### PR TITLE
Add ros-humble-perception

### PR DESCRIPTION
**Pull request by Taka-Kazu**
This PR adds `ros-humble-perception` and its dependent packages. 
The patches for `pluginlib` and `filters` are also included to avoid the compiler's internal error.

### CODE REVIEW

**Comment by Taka-Kazu** [comment]
GCC Internal Compiler error can happen when building `laster_filters` package.
```
-- Build files have been written to: /home/builder/aports/humble/ros-humble-laser-filters/abuild/build
[  6%] Building CXX object gtest/CMakeFiles/gtest_main.dir/src/gtest_main.cc.o
[ 12%] Building CXX object gtest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
[ 18%] Building CXX object CMakeFiles/laser_scan_filters.dir/src/laser_scan_filters.cpp.o
[ 25%] Linking CXX static library libgtest_main.a
[ 31%] Linking CXX static library libgtest.a
[ 37%] Linking CXX shared library liblaser_scan_filters.so
[ 37%] Built target gtest_main
[ 37%] Built target gtest
[ 43%] Building CXX object CMakeFiles/test_scan_filter_chain.dir/test/test_scan_filter_chain.cpp.o
[ 50%] Linking CXX executable test_scan_filter_chain
[ 50%] Built target laser_scan_filters
[ 56%] Building CXX object CMakeFiles/scan_to_cloud_filter_chain.dir/src/scan_to_cloud_filter_chain.cpp.o
[ 62%] Linking CXX executable scan_to_cloud_filter_chain
[ 62%] Built target test_scan_filter_chain
[ 68%] Building CXX object CMakeFiles/generic_laser_filter_node.dir/src/generic_laser_filter_node.cpp.o
[ 75%] Linking CXX executable generic_laser_filter_node
[ 75%] Built target scan_to_cloud_filter_chain
[ 81%] Building CXX object CMakeFiles/test_shadow_detector.dir/test/test_shadow_detector.cpp.o
[ 87%] Linking CXX executable test_shadow_detector
[ 87%] Built target generic_laser_filter_node
[ 93%] Building CXX object CMakeFiles/scan_to_scan_filter_chain.dir/src/scan_to_scan_filter_chain.cpp.o
[ 93%] Built target test_shadow_detector
during GIMPLE pass: strlen
In file included from /usr/ros/humble/include/pluginlib/pluginlib/class_loader.hpp:334,
                 from /usr/ros/humble/include/filters/filter_chain.hpp:39,
                 from /home/builder/aports/humble/ros-humble-laser-filters/abuild/src/laser_filters/src/scan_to_scan_filter_chain.cpp:40:
/usr/ros/humble/include/pluginlib/pluginlib/class_loader_imp.hpp: In function 'std::vector<std::__cxx11::basic_string<char> > pluginlib::ClassLoader<T>::getAllLibraryPathsToTry(const std::string&, const std::string&) [with T = filters::FilterBase<sensor_msgs::msg::LaserScan_<std::allocator<void> > >]':
/usr/ros/humble/include/pluginlib/pluginlib/class_loader_imp.hpp:375:35: internal compiler error: Segmentation fault
  375 | std::vector<std::string> NOINLINE ClassLoader<T>::getAllLibraryPathsToTry(
      |                                   ^~~~~~~~~~~~~~
0x1b51f37 internal_error(char const*, ...)
	???:0
Please submit a full bug report, with preprocessed source (by using -freport-bug).
Please include the complete backtrace with any bug report.
See <https://gitlab.alpinelinux.org/alpine/aports/-/issues> for instructions.
make[2]: *** [CMakeFiles/scan_to_scan_filter_chain.dir/build.make:76: CMakeFiles/scan_to_scan_filter_chain.dir/src/scan_to_scan_filter_chain.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:217: CMakeFiles/scan_to_scan_filter_chain.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
>>> ERROR: ros-humble-laser-filters: build failed
```

It seems that the error might be reduced by the patches for `pluginlib` and `filters`, but it still happens randomly.

**Comment by Taka-Kazu** [comment]
https://github.com/seqsense/aports-ros-experimental/pull/834#issuecomment-1571211425

I commented out all the logging macros in `pluginlib/include/pluginlib/class_loader_imp.hpp` in https://github.com/seqsense/aports-ros-experimental/pull/834/commits/9b47581349bb048bdb375e520a5353343a50c854. 
It should be revised after we have a newer alpine base image (with newer gcc).

**Review by at-wat — COMMENTED** [review]
https://github.com/seqsense/aports-ros-experimental/pull/889 
Locally tested that pluginlib error/warn logs can be restored

**Comment by at-wat** [comment]
Better to increment pkgrel of packages using pluginlib to test builds

**Comment by at-wat** [comment]
The gcc bug occurs on the buildfarm but doesn't occur on my local machine.

File: ros/humble.v3.17/ros-humble-pluginlib/workaround-gcc-segv.patch [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+   document.LoadFile(xml_file.c_str());
+   tinyxml2::XMLElement * config = document.RootElement();
+@@ -703,24 +703,24 @@ void ClassLoader<T>::processSingleXMLPluginFile(
+   while (library != NULL) {
+     const char* path = library->Attribute("path");
+     if (NULL == path) {
+-      RCUTILS_LOG_ERROR_NAMED("pluginlib.ClassLoader",
+-        "Attribute 'path' in 'library' tag is missing in %s.", xml_file.c_str());
++      RCUTILS_LOG_ERROR_NAMED("pluginlib.ClassLoader",
++        ("Attribute 'path' in 'library' tag is missing in ", xml_file + ".").c_str());
```
Comment (at-wat) [inline ros/humble.v3.17/ros-humble-pluginlib/workaround-gcc-segv.patch:346]:
```suggestion
+        ("Attribute 'path' in 'library' tag is missing in " + xml_file + ".").c_str());
```

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**